Worksheet B/Pseudocode.MD

```diff
@@ -0,0 +1,38 @@
+```
+
+
+Word Bank: Repulsive, Bubble, Men, Roasted, Sophisticated,
+Sturdy, Pretend, Nod, Group, Rat, Open, Dramatic, Etc…
+
+Set χ = 0
```

Comment:
Interesting use of a Greek letter for a variable name, but better to use something more descriptive